ios/FluentUI.Demo/FluentUI.Demo/ReadmeViewController.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+import FluentUI
+
+class ReadmeViewController: UIViewController {
+
+    let scrollView = UIScrollView()
+    let contentView = UIView()
+
+    let readme: String?
+
+    let readmeLabel: Label = {
+        let label = Label()
+        label.numberOfLines = 0
+        label.sizeToFit()
+        label.textColor = UIColor(dynamicColor: DynamicColor(light: ColorValue(0x000000),
+                                                             dark: ColorValue(0xFFFFFF)))
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        setupScrollViewConstraints()
+        setupReadmeLabelConstraints()
+        setReadmeLabelText()
+    }
+
+    private func setupReadmeLabelConstraints() {
+        contentView.addSubview(readmeLabel)
+        readmeLabel.centerXAnchor.constraint(equalTo: contentView.centerXAnchor).isActive = true
+        readmeLabel.topAnchor.constraint(equalTo: contentView.topAnchor, constant: Constants.topPadding).isActive = true
+        readmeLabel.widthAnchor.constraint(equalTo: contentView.widthAnchor, multiplier: Constants.widthMultiplier).isActive = true
+        readmeLabel.bottomAnchor.constraint(equalTo: contentView.bottomAnchor).isActive = true
+    }
+
+    private func setupScrollViewConstraints() {
+        scrollView.translatesAutoresizingMaskIntoConstraints = false
+        contentView.translatesAutoresizingMaskIntoConstraints = false
+
+        view.addSubview(scrollView)
+        scrollView.addSubview(contentView)
+
+        scrollView.centerXAnchor.constraint(equalTo: view.centerXAnchor).isActive = true
+        scrollView.widthAnchor.constraint(equalTo: view.widthAnchor).isActive = true
+        scrollView.topAnchor.constraint(equalTo: view.topAnchor).isActive = true
+        scrollView.bottomAnchor.constraint(equalTo: view.bottomAnchor, constant: Constants.bottomPadding).isActive = true
+
+        contentView.centerXAnchor.constraint(equalTo: scrollView.centerXAnchor).isActive = true
+        contentView.widthAnchor.constraint(equalTo: scrollView.widthAnchor).isActive = true
+        contentView.topAnchor.constraint(equalTo: scrollView.topAnchor).isActive = true
+        contentView.bottomAnchor.constraint(equalTo: scrollView.bottomAnchor).isActive = true
+    }
+
+    struct Constants {
+        static let topPadding: CGFloat = 35
+        static let bottomPadding: CGFloat = -25
+        static let widthMultiplier: CGFloat = 0.9
+        static let popoverWidth: CGFloat = 400
```

Comment:
Have you tried this out in iPad, esp in landscape mode? Not a big deal for an FHL project, just something to consider longer term

Reply:
I didn't do thorough testing on iPad but these dimensions seem ok. I'll be happy to tweak them since I didn't really put much thought into these values, I just went with what felt right to the eye.

<img width="857" alt="Screen Shot 2022-09-23 at 2 13 23 PM" src="https://user-images.githubusercontent.com/106181067/192057651-c842b7c0-42c9-418a-85d4-5bf705ccbd5a.png">